driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -1259,6 +1263,28 @@ synchronized void init() {
             try {
                 while (true) {
                     try {
+
+                        // create debouncers - at this stage, they are not running yet
+                        QueryOptions queryOptions = configuration.getQueryOptions();
+                        this.nodeListRefreshRequestDebouncer = new EventDebouncer<NodeListRefreshRequest>(
```

Comment:
I was precisely worried about this request showing up one day :) because this counter is not meant to be 100% accurate (see my previous comments). I'm not against the idea of the gauge, but we should put a warning about accuracy on it (it's not atomically updated in respect to the queue so it could temporarily be above the maximum or below zero).